File: Listas antigas em C/Lista 1/Questão 14 Lista 1.c
/*
Questão 14 Lista 1
Faça um algoritmo que calcule e imprima o An de uma P.A. (Progressão Aritmética), segundo a fórmula: An = a1 + (n-1) * r.
*/
#include <stdio.h>  
int main(void) {
	int a1, r, n, An;

	printf("digite a1, r e n\n");
	scanf("%d",&a1);
	scanf("%d",&r);
	scanf("%d",&n);
	An = a1+(n-1)*r;
	printf("An = %d", An);
	return 0; 
}

File: Lista 1/Questão 14 Lista 1.py
"""
Questão 14 Lista 1
Faça um algoritmo que calcule e imprima o An de uma P.A. (Progressão Aritmética), segundo a fórmula: An = a1 + (n-1) * r.
"""
a1 = int(input("Digite o valor de a1: "))
n = int(input("Digite o valor de n: "))
r = int(input("Digite o valor de r: "))
An = a1+(n-1)*r
print(f"An = {An}")